src/components/forms/forms.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  getFormsDataThunk,
+  deleteFormSchemaThunk,
+  saveFormDataThunk
+} from './../../redux/forms-Reducer';
+import style from './forms.module.css';
+
+function Forms(props) {
+  const [state, setState] = useState({});
+
+  if (!props.checked) {
+    props.getFormsDataThunk();
+  } else if (!state.forms) setState({ forms: [...props.forms] });
+
+  const onSubmit = (e, key) => {
```

Comment:
```suggestion
  const handleSubmit = (e, key) => {
```
-> all component actions handler, need to be prefixed with handleX, handleActionName -> handleClick, handleBlur
-> indeed, onSubmit and all onX names are also used, but they are only used when you send them as props. So, when you render a Component you'll pass in props with **on** prefix, but when you write actions inside the component, you'll use **handle**

props:
```
<MyForm
  onSubmit={handleSubmit}
  onClick={handleClick}
  {...otherProps}
/>
```

component actions:
```
const { onSubmit, onClick } = props;

const handleSubmit = () = {
  doCustomStuff();
  wait();
  onSubmit && onSubmit();
};

const handleClick = () = {};
```